### PR TITLE
Issue 1025

### PR DESCRIPTION
**Pull request by edytagarbarz**
# I have made things!

## Checklist

- [X] I have double checked that there are no unrelated changes in this pull request (old patches, accidental config files, etc)
- [X] I have created at least one test case for the changes I have made
- [X] I have updated the documentation for the changes I have made
- [ ] I have added my changes to the `CHANGELOG.md`

## Related issues

Refs #1025 

This is a solution to one part of the issue #1025, responsible for restricting usage of multiline strings to docstrigns and assignments.

🙏 Please, if you or your company is finding wemake-python-styleguide valuable, help us sustain the project by sponsoring it transparently on https://opencollective.com/wemake-python-styleguide. As a thank you, your profile/company logo will be added to our main README which receives hundreds of unique visitors per day.

### CODE REVIEW

File: tests/test_visitors/test_tokenize/test_statements/test_multiline_string.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+
+wrong_compare = '''
+a = 'abc'
+if a > """ab
+cd""":
+    return 1
+'''
+
+wrong_function_call = '''
```
Comment (sobolevn) [inline tests/test_visitors/test_tokenize/test_statements/test_multiline_string.py:46]: What about this code?

```python
"""ab
cd""".split('\n')
```

?

File: wemake_python_styleguide/violations/best_practices.py [hunk 2]
```diff
@@ -2106,3 +2108,49 @@ class ShebangViolation(SimpleViolation):
...
+    Multiline strings are only allowed in docstrings
+    or assignments to variables.
+
+    Reasoning:
+        Direct usage of multiline strings is not readable.
+        One should not depend on the current indentation,
+        e.g. in comparisons or function calls.
+
+    Solution:
+    Assign a multiline string to a variable.
```
Comment (sobolevn) [inline wemake_python_styleguide/violations/best_practices.py:2127]: Missing indentation.

File: wemake_python_styleguide/violations/best_practices.py [hunk 3]
```diff
@@ -2106,3 +2108,49 @@ class ShebangViolation(SimpleViolation):
...
+
+    Example::
+
+        #Correct:
```
Comment (sobolevn) [inline wemake_python_styleguide/violations/best_practices.py:2131]: `# Correct:`

File: wemake_python_styleguide/violations/best_practices.py [hunk 4]
```diff
@@ -2106,3 +2108,49 @@ class ShebangViolation(SimpleViolation):
...
+        multiline = """
+            abc
+            abc
+        """
+        function(multiline)
```
Comment (sobolevn) [inline wemake_python_styleguide/violations/best_practices.py:2136]: We don't need this code.

File: wemake_python_styleguide/violations/best_practices.py [hunk 5]
```diff
@@ -2106,3 +2108,49 @@ class ShebangViolation(SimpleViolation):
...
+        if var == multiline:
+            return 1
+
+        #Wrong:
```
Comment (sobolevn) [inline wemake_python_styleguide/violations/best_practices.py:2140]: `# Wrong:`

File: wemake_python_styleguide/violations/best_practices.py [hunk 6]
```diff
@@ -2106,3 +2108,49 @@ class ShebangViolation(SimpleViolation):
...
+        function("""
```
Comment (sobolevn) [inline wemake_python_styleguide/violations/best_practices.py:2141]: Just one example is fine.

File: tests/test_visitors/test_tokenize/test_statements/test_multiline_string.py (hunk 1 above)
Comment (edytagarbarz) [reply tests/test_visitors/test_tokenize/test_statements/test_multiline_string.py:46]: Thanks a lot for this catch!

**Review by sobolevn — APPROVED** [review]

**Pull request merged**